### ISSUE TITLE
find-sec-bugs always claims "The following classes needed for analysis were missing" for enums 

### ISSUE THREAD

**Issue by mkienenb** [find-sec-bugs/find-sec-bugs#176]
While investigating issue #175, I noticed that part of the problem was that all enum types were reported as missing classes.   I eventually reduced one specific class down to this simple example and was still able to reproduce the error using a snapshot built from 2016-02-15.
```
package com.xyz.ebpp.business;

public enum CheckLoginLogConsistencyForCloudServiceUsers {
    HTTP_REQUEST
}
```

Generates:
```
compile:
    [javac] /home/mkienenb/workspaces/xyz-keplar/Project/building/build-find-sec-bugs-test.xml:103: warning: 'includeantruntime' was not set, defaulting to build.sysclasspath=last; set to false for repeatable builds
    [javac] Compiling 1 source file to /home/mkienenb/workspaces/xyz-keplar/Project/WEB-INF/classes-test-find-sec-bugs

java_version_greater_than_1.7:

findbugs-main:
 [findbugs] Executing findbugs FindBugsTask from ant task
 [findbugs] Running FindBugs...
 [findbugs] Feb 16, 2016 12:18:58 PM com.h3xstream.findsecbugs.injection.custom.CustomInjectionSource getInjectionSourceProperties
 [findbugs] INFO: The optional configuration for additional injection sources (com/h3xstream/findsecbugs/injection/custom/CustomInjectionSource.properties) was not found. This message can be ignored if no custom API are intended to be configured. For more info: http://find-sec-bugs.github.io/bugs.htm#CUSTOM_INJECTION
 [findbugs] The following classes needed for analysis were missing:
 [findbugs]   com.xyz.ebpp.business.CheckLoginLogConsistencyForCloudServiceUsers[]
 [findbugs] Missing classes: 1
 [findbugs] Calculating exit code...
 [findbugs] Setting 'missing class' flag (2)
 [findbugs] Exit code set to: 2
 [findbugs] Java Result: 2
 [findbugs] Classes needed for analysis were missing
 [findbugs] Output saved to ...
```

**Comment by mkienenb** [comment]
Verified that this was fixed by pull request #173

**Comment by h3xstream** [comment]
@mkienenb  Thanks for the precision
